assignment/assignment-bundles/resources/assignment.properties

```diff
@@ -350,7 +350,7 @@ plesuse0 = ''{0}'' is invalid. Please use correct grade string.
 
 plesuse1 = Please use a number for the grade field.
 
-plesuse2 = Please use only one decimal place for the grade field.
+plesuse2 = Please use only {0} decimal(s) place for the grade field.
```

Comment:
Minor grammar correction, this should read:

Please use only {0} decimal place(s) for the grade field.